### PR TITLE
Gracefully imports dcc on Dash 1.x and 2.x

### PR DESCRIPTION
**Pull request by antoinerg**
cc @alexcjohnson

### CODE REVIEW

**Comment by antoinerg** [comment]
@alexcjohnson I don't have admin access to CircleCI so I couldn't add this repo as a project! Could you please add it? :pray:

**Comment by alexcjohnson** [comment]
https://app.circleci.com/pipelines/github/plotly/dash-enterprise-auth

File: dash_enterprise_auth/__init__.py [hunk 1]
```diff
@@ -11,7 +11,11 @@
 import functools as _ft
 import json as _json
 
-import dash_core_components as _dcc
+import dash
+if hasattr(dash, "dcc"):
+    _dcc = dash.dcc
```
Comment (alexcjohnson) [inline dash_enterprise_auth/__init__.py:16]:
```suggestion
import dash as _dash
if hasattr(_dash, "dcc"):
    _dcc = _dash.dcc
```
Just for consistency, as this module seems to want to be careful what its exports look like.

**Review by alexcjohnson — APPROVED** [review]
💃 LGTM, just one minor comment.

**Pull request merged**